ontopy/graph.py

```diff
@@ -691,6 +691,27 @@ def get_node_attrs(self, name, nodeattrs, attrs):
         kwargs.update(attrs)
         return kwargs
 
+    def _relation_styles(self, entity, relations, rels, name):
+        """Helper function that returns the styles of the relations
+        to be used.
+        """
```

Comment:
This is a great method abstraction. It also provides an excellent opportunity to describe the differences between `relations` and `rels`. Or indeed, just describe all of the arguments in the doc-string :)